wcivf/apps/elections/import_helpers.py

```diff
@@ -280,6 +281,21 @@ def set_organisation_type(self, ballot):
             ]
             ballot.post.save()
 
+    def set_division_type(self, ballot):
+        """
+        Attempts to set the division_type field from EveryElection
+        """
+        if ballot.post.division_type and not self.force_update:
+            return
+
+        ee_data = self.ee_helper.get_data(ballot.ballot_paper_id)
+
+        if not ee_data or not ee_data["division"]:
+            return
+
+        ballot.post.division_type = ee_data["division"].get("division_type")
```

Comment:
Will this raise a `ValueError` if the type isn't in `DIVISION_TYPE_CHOICES`? I think it would be useful to do this, to make sure we don't forget to update the mapping when / if we add more types in EE

Reply:
TIL;

No, it doesn't by default. But by adding a call to `ballot.full_clean()` before the save, then yes a `ValidationError` is raised if the value isn't in `DIVISION_TYPE_CHOICES`. So I think that would help with the above issue, to make sure that if a new division_type is added in EE it is also updated here.